pom.xml

```diff
@@ -550,6 +555,44 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.checkerframework</groupId>
+      <artifactId>checker-qual</artifactId>
+      <version>2.6.0</version>
```

Comment:
Why are you using version 2.6.0 of the Checker Framework? The most recent version is 2.8.1.